### DISCUSSION TITLE
Updates go brrrr

### DISCUSSION THREAD

**Discussion by 3kh0** [3kh0/ext-remover#752] [Announcements]
Hello fellow gamers!

I have been updating this repo with all the new exploits. Many thanks to @Brandon421-ops for most of them. What a epic person.

Have fun!

**Comment by weenslurper** [comment]
the uBlock Origin one would be so good if could figure out how it works LMAO

**Reply by weenslurper** [reply]
maybe you can understand the instructions you called confusing because you wrote them, idk just a thought

**Reply by Xty21223** [reply]
However, I will try to interpret, it and then pull a request for an instruction update.

**Comment by MechaXYZ** [comment]
gaming

**Comment by Xty21223** [comment]
This is awsome!
